diff --git a/Cargo.lock b/Cargo.lock
index 9833624..d4d3171 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -307,8 +307,6 @@ checksum = "52ff2c0fe9bc6cb6b14a0592c2ff4fa9ceb83eea9db979b0487cd054946a2b8f"
 [[package]]
 name = "libdeflate-sys"
 version = "1.26.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "6949d73714ba8c32d2757405b89c94e427f64f078a4041debe07e1b8e9e850e9"
 dependencies = [
  "cc",
  "pkg-config",
@@ -317,8 +315,6 @@ dependencies = [
 [[package]]
 name = "libdeflater"
 version = "1.26.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "0b72b274104f747cb65358c918e38c6cd4a69937937a36c8ad3cfd516c9c471e"
 dependencies = [
  "libdeflate-sys",
 ]
diff --git a/src/cli.rs b/src/cli.rs
index 77e5f1b..3e5cb29 100644
--- a/src/cli.rs
+++ b/src/cli.rs
@@ -276,15 +276,15 @@ be disabled at level 5 or above.")
         )
         .arg(
             Arg::new("compression")
-                .help("Deflate compression level (0-12)")
+                .help("Deflate compression level (0-14)")
                 .long_help("\
-Deflate compression level (0-12) for main compression trials. The levels here are defined \
-by the libdeflate compression library.
+Deflate compression level (0-14) for main compression trials. The levels here are defined \
+by the libdeflate compression library; 13 and 14 come from a patch that compresses further.
 
 The default value depends on the optimization level preset.")
                 .long("zc")
                 .value_name("level")
-                .value_parser(0..=12)
+                .value_parser(0..=14)
                 .conflicts_with("zopfli"),
         )
         .arg(
